moroScript.jsx

```diff
@@ -207,9 +207,9 @@
       var Definition = React.createClass({
         render: function() {
           return (
-            <div>
+            <div className="ui vertical segment">
               <h2>
-                {this.props.moroword}
+                {_.join(this.props.moroword, ", ")}
```

Comment:
Now `this.props.moroword` is an array rather than a string. Use `lodash`'s `_.join` method to turn the array into a string once again when we render it.

Otherwise react renders the array as a bunch of `span` tags which results in them being concatenated.